.github/workflows/test.yml

```diff
@@ -31,4 +31,7 @@ jobs:
         go-version: 1.16
 
     - name: Test
+      env:
+        tf_token: ${{ secrets.OS_TF_TOKEN }}
+        tf_organization: ryanwholey
```

Comment:
what do we think about setting up a scoop OS cloud org?

Reply:
What's the procedure for maintaining membership? For distributing tokens to GH?

This is almost certainly the right move but making it both free and automated may not be possible. So we just need some internal docs on how it's used.

Also, OSS = Open Source Software, OS = Operating System

Reply:
yeah great question! I was thinking the best way to maintain this would be to create a TFE workspace and manage the TF cloud organization user management via Okta, all engineering has access, and then a machine user to create a github token, which can be placed via terraform as well.

Getting this off the ground though, I'd be interested in merging with my tfc account, or just creating an initial scoop one by hand, because it gives us better coverage as we get closer to the big move

Reply:
Okta will cost $$. Create a `takescoop-oss` org. We'll have to manage it manually.